modules/decode.py

```diff
@@ -163,4 +170,9 @@ def decode_astropix2_hits(self, list_hits: list, i, file):
             file.write(f"{i}\t")
             file.write( f"{wrong_id}\t {wrong_payload}\t {location}\t{'Col' if col else 'Row'}\t{timestamp}\t {tot_msb}\t{tot_lsb} \t {tot_total} \t {(tot_total * self.sampleclock_period_ns)/1000.0} \n"
             )
+            ### THIS IS NEW CODE Autumn on Jun 14 2022. Added in an option             
```

Comment:
@Narg3000  We can actually use `git annotate` on the command line (or check the web interface) to see who edited the code when, so it is more important to put in comments that describe what you are doing (if that's not clear from the code) or why.